docs/POOL_EXAMPLES_ETH.md

```diff
@@ -128,6 +128,7 @@ The servers are listed in alphabetical order. To get best results reorder them f
 | Pool Name | Pool Homepage | Details about connection |
 | --------- | ------------- | - |
 | [2miners.com](#2minerscom) | <https://2miners.com/> | <https://eth.2miners.com/en/help> |
+| [pool.binance.com](#poolbinancecom) | <https://pool.binance.com/> | <https://www.binance.com/en/support/faq/f77a002dca084b22ab2e13ccf2f13f18> |
```

Comment:
| [pool.binance.com](#poolbinancecom) | <https://pool.binance.com/> | <https://www.binance.com/en/support/faq/f77a002dca084b22ab2e13ccf2f13f18> |

Reply:
I don’t know what you want me to write